website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -44,6 +44,8 @@ For example, you would use the following command if you use Snowflake:
 pip install dbt-core dbt-snowflake
 ```
 
+For the time being, we have maintained install-time dependencies to avoid breaking existing scripts in surprising ways; `pip install dbt-snowflake` will continue to install `dbt-core` implicitly. Given that we may remove this implicit dependency in future versions, we strongly encourage you to update install scripts **now**.
```

Comment:
Yes, good clarification — for the time being, `pip install dbt-snowflake` will continue to install the latest versions of both `dbt-snowflake` and `dbt-core`. In the future, it will no longer install `dbt-core`, so users should start installing it directly.

Reply:
K I can add a clarification here